wafflehouse.gemspec

```diff
@@ -28,4 +28,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.12"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "minitest", "~> 5.0"
+  spec.add_dependency 'restforce'
```

Comment:
Please specify some kind of version here. This would prevent us from upgrading to an incompatible version of restforce.